cf-custom-resources/lib/backlog-per-task-calculator.js

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+"use strict";
+
+const aws = require("aws-sdk");
+
+// AWS Clients that are overriden in tests.
+let ecs, sqs;
+
+/**
+ * This lambda function calculates the backlog of SQS messages per running ECS tasks,
+ * and writes the metric to CloudWatch.
+ */
+exports.handler = async (event, context) => {
+  setupClients();
+  try {
+    const runningCount = await getRunningTaskCount(process.env.CLUSTER_NAME, process.env.SERVICE_NAME);
```

Comment:
Where do we define `process`?

Reply:
It's a global module that's always available! https://nodejs.org/en/knowledge/getting-started/the-process-module/